java/src/main/java/com/google/cloud/dataproc/templates/jdbc/JDBCToGCS.java

```diff
@@ -64,16 +71,48 @@ public JDBCToGCS() {
 
   @Override
   public void runTemplate() {
+    if (StringUtils.isAllBlank(jdbcSQL) && StringUtils.isAllBlank(jdbcSQLFile)) {
+      LOGGER.error(
+          "Required parameter not passed. Either {} or {} parameter must be provided. ",
+          JDBC_TO_GCS_SQL,
+          JDBC_TO_GCS_SQL_FILE);
+      throw new IllegalArgumentException(
+          "Required parameters for JDBCToGCS not passed. "
+              + "Set mandatory parameter for JDBCToGCS template "
+              + "in resources/conf/template.properties file or at runtime. "
+              + "Refer to jdbc/README.md for more instructions.");
+    } else if (StringUtils.isAllBlank(jdbcSQL)) {
```

Comment:
The condition here is cryptic.
Instead please use: StringUtils.isNotBlank(jdbcSQLFile)

Reply:
Moved all validations from driver file (JDBCToGCS.java) to *Config.java file. Re-wrote all the validations with Jakarta bean validation. Please review.